CovertInfiltration/Config/XComInfiltration.ini

```diff
@@ -62,4 +12,118 @@ RANKS_DETER[4] = 8
+		
++InfilModifiers=(Item="AlienHunterRifle_CV",	HoursAdded=12, RiskReductionPercent=7)
++InfilModifiers=(Item="AlienHunterRifle_MG",	HoursAdded=12, RiskReductionPercent=7)
++InfilModifiers=(Item="AlienHunterRifle_BM",	HoursAdded=12, RiskReductionPercent=7)
+	
++InfilModifiers=(Item="AlienHunterPistol_CV",	HoursAdded=2, RiskReductionPercent=7)
++InfilModifiers=(Item="AlienHunterPistol_MG",	HoursAdded=2, RiskReductionPercent=7)
++InfilModifiers=(Item="AlienHunterPistol_BM",	HoursAdded=2, RiskReductionPercent=7)
+	
++InfilModifiers=(Item="AlienHunterAxe_CV",	HoursAdded=3, RiskReductionPercent=7)
++InfilModifiers=(Item="AlienHunterAxe_MG",	HoursAdded=3, RiskReductionPercent=7)
++InfilModifiers=(Item="AlienHunterAxe_BM",	HoursAdded=3, RiskReductionPercent=7)
+
++InfilModifiers=(Item="Bullpup_CV",			HoursAdded=10, RiskReductionPercent=1)
++InfilModifiers=(Item="Bullpup_MG",			HoursAdded=10, RiskReductionPercent=3)
++InfilModifiers=(Item="Bullpup_BM",			HoursAdded=10, RiskReductionPercent=4)
+	
++InfilModifiers=(Item="VektorRifle_CV",		HoursAdded=12, RiskReductionPercent=1)
++InfilModifiers=(Item="VektorRifle_MG",		HoursAdded=12, RiskReductionPercent=3)
++InfilModifiers=(Item="VektorRifle_BM",		HoursAdded=12, RiskReductionPercent=4)
+	
++InfilModifiers=(Item="ShardGauntlet_CV",	HoursAdded=8, RiskReductionPercent=3)
++InfilModifiers=(Item="ShardGauntlet_MG",	HoursAdded=8, RiskReductionPercent=4)
++InfilModifiers=(Item="ShardGauntlet_BM",	HoursAdded=8, RiskReductionPercent=5)
+	
++InfilModifiers=(Item="Pistol_CV",	HoursAdded=2)
++InfilModifiers=(Item="Pistol_MG",	HoursAdded=2)
++InfilModifiers=(Item="Pistol_BM",	HoursAdded=2)
++InfilModifiers=(Item="Sidearm_CV", HoursAdded=2)
++InfilModifiers=(Item="Sidearm_MG", HoursAdded=2)
++InfilModifiers=(Item="Sidearm_BM", HoursAdded=2)
+	
++InfilModifiers=(Item="GrenadeLauncher_CV", HoursAdded=10)
++InfilModifiers=(Item="GrenadeLauncher_MG", HoursAdded=10)
+	
++InfilModifiers=(Item="Gremlin_CV",		HoursAdded=4)
++InfilModifiers=(Item="Gremlin_MG",		HoursAdded=4)
++InfilModifiers=(Item="Gremlin_BM",		HoursAdded=4)
++InfilModifiers=(Item="SparkBit_CV",	HoursAdded=4)
++InfilModifiers=(Item="SparkBit_MG",	HoursAdded=4)
++InfilModifiers=(Item="SparkBit_BM",	HoursAdded=4)
+	
++InfilModifiers=(Item="Sword_CV",		HoursAdded=3)
++InfilModifiers=(Item="Sword_MG",		HoursAdded=3)
++InfilModifiers=(Item="Sword_BM",		HoursAdded=3)
++InfilModifiers=(Item="WristBlade_CV",	HoursAdded=3)
++InfilModifiers=(Item="WristBlade_MG",	HoursAdded=3)
++InfilModifiers=(Item="WristBlade_BM",	HoursAdded=3)
+
++InfilModifiers=(Item="PsiAmp_CV", HoursAdded=-6)
++InfilModifiers=(Item="PsiAmp_MG", HoursAdded=-6)
++InfilModifiers=(Item="PsiAmp_BM", HoursAdded=-6)
+
++InfilModifiers=(Item="HoloProjector",	HoursAdded=0, RiskReductionPercent=0, MultiplierCategory="weapon",	InfilMultiplier=0.5)
++InfilModifiers=(Item="AdaptiveCamo",	HoursAdded=0, RiskReductionPercent=0, MultiplierCategory="armor",	InfilMultiplier=0.5)
```

Comment:
Get rid of `HoursAdded=0, RiskReductionPercent=0`, that's the default values anyway

Reply:
Again, I think the current way is clearer to a non-programmer.